packages/ragbits-document-search/src/ragbits/document_search/retrieval/rephrasers/llm.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional
+
+from ragbits.core.llms import get_llm
+from ragbits.core.llms.base import LLM, LLMOptions
+from ragbits.core.prompt import Prompt
+from ragbits.document_search.retrieval.rephrasers.base import QueryRephraser
+from ragbits.document_search.retrieval.rephrasers.prompt_query_rephraser import QueryRephraserPrompt, _PromptInput
+
+
+class LLMQueryRephraser(QueryRephraser):
+    """A rephraser class that uses a LLM to rephrase queries."""
+
+    def __init__(self, llm: LLM):
```

Comment:
There should also be a `prompt` parameter, that would tell us what rephrasing strategy we want to use by default.

Reply:
ok, fixed